README.md

```diff
@@ -32,45 +32,68 @@ Bambi requires working versions of ArviZ, formulae, NumPy, pandas and PyMC. Depe
 In the following two examples we assume the following basic setup
 
 ```python
+import arviz as az
 import bambi as bmb
 import numpy as np
 import pandas as pd
-
-data = pd.DataFrame({
-    "y": np.random.normal(size=50),
-    "g": np.random.choice(["Yes", "No"], size=50),
-    "x1": np.random.normal(size=50),
-    "x2": np.random.normal(size=50)
-})
 ```
 
 ### Linear regression
 
+A simple fixed effect model is shown in the example below.
+
 ```python
-model = bmb.Model("y ~ x1 + x2", data)
-fitted = model.fit()
+#### Read in a database from the package content
+data = bmb.load_data("sleepstudy")
```

Comment:
if data is loaded it should be outputted (`data.head()`) so that users know what the format is.